schemas/session-replay/mobile/wireframe-clip-schema.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json-schema.org/draft-07/schema",
   "$id": "session-replay/mobile/wireframe-clip-schema.json",
   "title": "WireframeClip",
-  "type": "object",
+  "type": ["object", "null"],
```

Comment:
This will create issues on our end and we will have to update the schema generators. I want to challenge this as we never used this approach before. Is this a correct way of declaring types in a JSON schema ?